app/views/contact/govuk/accessible_format_requests/form.html.erb

```diff
@@ -6,19 +6,22 @@
     <%= render "govuk_publishing_components/components/radio", {
       name: question_key,
       heading: question_caption,
-      items: question_options
+      items: question_options,
+      error_message: flash["input_errors"]&.[](question_key)
```

Comment:
I haven't come across this syntax before - specifically the `[](question_key)` bit. What is that doing?

Reply:
It's a rather clunky way of accessing the key in the `FlashHash`, which doesn't seem to allow direct `[question_key]` access at a second level
https://api.rubyonrails.org/classes/ActionDispatch/Flash/FlashHash.html#method-i-5B-5D